plugin/eqdetect/source/dextool/plugin/eqdetect/subfolder/codegenerator.d

```diff
@@ -51,4 +51,48 @@ class SnippetFinder{
         temp = temp ~ content[mutation.offset_end .. content.length];
         return temp;
     }
+
+    @trusted static auto generateKlee(string[] params, string source_name, string mutant_name, string function_name){
+        import std.stdio;
+        import std.format;
+        //string code;
+        import dsrcgen.c;
```

Comment:
tip: there is a cpp module too :)

Reply:
Yes, we want to use it in the future. There are currently a lot of code written only to get something to work with KLEE that needs changing or completely removal in the future.